packages/teams-js/src/public/runtime.ts

```diff
@@ -7,6 +7,7 @@ export interface IRuntime {
   readonly apiVersion: number;
   readonly isLegacyTeams?: boolean;
   readonly supports: {
+    readonly adaptiveCard?: {};
```

Comment:
curious - what is this representing?

Reply:
The intention is that it will just be something that hosts use to pass down the version of adaptive cards that they support. While I'd like for there to never be different versions, in practice (especially in the future) we'll have hosts who run different versions, even if it's just for A/B testing new versions of the dialog. Having this split apart from the main part of the adaptive card dialogs lets us keep supporting that flow for Teams while also enforcing something like a minimum viable AC version in the new capability

Reply:
Sorry, I will be super clear just to make sure we're on the same page: this *is* a new capability, it's just that all it provides is the ability to check what version of adaptive cards is supported: 
![image](https://user-images.githubusercontent.com/18446904/181820885-b9803225-2dcd-4ce7-b2c8-74e6b0497817.png)

Reply:
We're then saying that the new adaptive card support in dialog needs this to also be implemented, and will be using it to both let developers check what the currently supported version of adaptive cards is in the dialog on that host. We'll also use it internally to validate that the host is meeting a minimum acceptable version of the AC schema.